ui/scripts/docs.js

```diff
@@ -1008,7 +1008,7 @@ cloudStack.docs = {
     },
     // Add volume
     helpVolumeName: {
-        desc: 'Give the volume a unique name so you can find it later.',
+        desc: 'Give a unique volume name. If it is not provided, a name will be generated randomly.',
```

Comment:
Ah, so in the present state you do not consider this worth a :-1: , I read from your comment.